util/users.py

```diff
@@ -26,3 +26,5 @@ async def register_user(form, db):
 
     # return user_id if success
     return user_id
+
+async def update_item():
```

Comment:
Suggestion:

- flow of update operation:
1. pull existing user data (verify that user exists at this point else throw 404)
2. replace old data with new data IF provided (aka only update if input given) look into `dict.update`
3. reinsert data into DB with `column.update(query)` where `query` = user identifier